### PR TITLE
Move mysqlrouter `/var/run` to `/run`

### PR DESCRIPTION
**Pull request by carlcsaposs-canonical**
FHS 3.0 (released in 2015) moved `/var/run` to `/run`
https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

- Update mysql packages from 8.0.32 to 8.0.33
- Install mysqlsh from PPA

### CODE REVIEW

**Comment by carlcsaposs-canonical** [comment]
Ubuntu 18.04 (and potentially earlier) uses FHS 3.0

**Review by paulomach — APPROVED** [review]

**Review by taurus-forever — APPROVED** [review]
Hint, try to unpack this PR snap into OCI to make sure no surprises there. LGTM.

**Comment by taurus-forever** [comment]
LGTM for version: '8.0.33-0'. Time to go forward!

**Comment by carlcsaposs-canonical** [comment]
> LGTM for version: '8.0.33-0'. Time to go forward!

mysqlsh is still 8.0.32

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -20,8 +20,10 @@ package-repositories:
       - main
     suites:
       - jammy
-    key-id: 630239CC130E1A7FD81A27B140976EAF437D05B5
+    key-id: F6ECB3762474EDA9D21B7022871920D1991BC93C
     url: http://security.ubuntu.com/ubuntu/
```
Comment (carlcsaposs-canonical) [inline snap/snapcraft.yaml:24]: Context: https://chat.canonical.com/canonical/pl/y6asf86jg7g4iq5ig8ehpd3aro

Comment (carlcsaposs-canonical) [reply snap/snapcraft.yaml:24]: `630239CC130E1A7FD81A27B140976EAF437D05B5` is an older version of the key from 2004 retrieved from http://security.ubuntu.com/ubuntu/project/ubuntu-archive-keyring.gpg

`F6ECB3762474EDA9D21B7022871920D1991BC93C` comes from `/etc/apt/trusted.gpg.d/ubuntu-keyring-2018-archive.gpg` on any Ubuntu machine and from https://wiki.ubuntu.com/SecurityTeam/FAQ#GPG_Keys_used_by_Ubuntu

**Pull request merged**